src/components/SearchBar.js

```diff
@@ -7,15 +7,21 @@ import { defaultTo } from 'lodash/fp';
 export default function SearchBar(props) {
   const { handleSearchChange } = props;
   const searchTerms = useRef('');
+
+  const width = props.width ? props.width : '100%';
```

Comment:
This seems kind of clunky if we wanted to update multiple style features in the future - Can I recommend we change this overall code to be more like:

```suggestion
  const style = {
    width: '100%',
    border: '1px solid #cecece',
    backgroundColor: '#f3f6f7',
    borderRadius: '5px',
    height: '4rem',
    paddingLeft: '2%',
    fontFamily: 'Montserrat',
    fontSize: '1.5rem',
    ...props.style
 }
```

Then in the section in AdminManageUser where SearchBar is instantiated, we update it to

```suggestion
  h(SearchBar, {
    ...
    style: {
      width: '60%',
      margin: '0 3% 0 0'
    }
  })
```

This will allow us to override all of the styles in the search bar without having to extract each one directly first

Reply:
this is a much smarter solution! thank you